packages/tools/create-kadena-app/scripts/generate-test.mjs

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env zx
+// This script is used to generate projects for every supported template and test them.
+
+import 'zx/globals';
+
+import { rm } from 'fs/promises';
+
+// the default log uses stderr for write output of commands, even for commands that do not generate errors or warnings.
+// This can cause Rush to treat everything as a warning message. This function changes this behavior.
+$.log = (entry) => {
```

Comment:
I an unexpected behaviour in the default log function. 
I have created an issue on the repo. if they change the behaviour we can remove this function
https://github.com/google/zx/issues/623